beps/bep_0003.rst

```diff
@@ -87,70 +92,153 @@ announce
 
 info
   This maps to a dictionary, with keys described below.
+  
+``piece layers``
```

Comment:
A high level comment on this overall structure: There's going to be an overlap in time where there are torrent files which support both new and old style data, and new style clients can connect to either type, then after a transition the old-style data should be dropped. A bit of redundancy between the old and new data isn't terribly problematic. What I'd like to see is that the new-style data all be included in a bencoded string under a single key in the info dict, so that its value is committed to by the old style data, and after a transition all other keys in the info dict are dropped. When new peers connect to each other they should specify the new algorithm hash of just the new data and communicate based on that, and they'll need to use a slightly extended peer protocol to be able to communicate paths to the root of individual chunks. 

I mostly agree with the new keys as presented here, but think they should be moved into the new dict.